rotkehlchen/chain/ethereum/node_inquirer.py

```diff
@@ -277,7 +277,10 @@ def assure_curve_protocol_cache_is_queried(self) -> bool:
         2. Queries information about curve pools' addresses, lp tokens and used coins
         3. Saves queried information in the cache in globaldb
         """
-        if should_update_protocol_cache(GeneralCacheType.CURVE_LP_TOKENS) is False:
+        if (
```

Comment:
You added a new argument and you have not explained what it's supposed to do. Please adjust the docstring.

I guess if it's true then cache is always requeried? Why not specify the name better then? `always_refresh_cache`?

Reply:
You still did not adjust the docstring though :sweat_smile: